transport-native-kqueue/src/main/java/io/netty/channel/kqueue/Native.java

```diff
@@ -110,6 +112,8 @@ public void run() {
     private static final int CONNECT_RESUME_ON_READ_WRITE = connectResumeOnReadWrite();
     private static final int CONNECT_DATA_IDEMPOTENT = connectDataIdempotent();
     static final int CONNECT_TCP_FASTOPEN = CONNECT_RESUME_ON_READ_WRITE | CONNECT_DATA_IDEMPOTENT;
+    static final boolean IS_SUPPORTING_TCP_FASTOPEN_CLIENT = fastOpenClient() == 1;
+    static final boolean IS_SUPPORTING_TCP_FASTOPEN_SERVER = fastOpenServer() == 1;
```

Comment:
as these native calls might fail with a runtime exception I wonder if it would be better to log in case of an error and just set the static boolean fields to false. Otherwise it will blowup and the app cant be started.